### PR TITLE
Story

### PR DESCRIPTION
**Pull request by ailindoyle**
Portfolio website build one (finished 01/03/2018).

Includes 4 pages for Home, About, Portfolio and Contact.

Has not been cross browser tested due to problem with local server.

Needs review.

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            </div>
+        </div>
+    </div>
+    <div class="logo">
+        <a href="home.html"><img src="images/CD-logo.png" alt="Logo"/></a>
+    </div>
+</header>
+<div id="hero">
+    <div class="main">
+        <div class="container">
```
Comment (charliecog) [inline home.html]: you wont need so many divs - just use additional classes instead

File: home.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                <div class="hero-box description">
+                    <h1>Caitlin Doyle</h1>
+                    <h3>Web Developer</h3>
+                </div>
+        </div>
+    </div>
+</div>
+<div id="summary">
```
Comment (charliecog) [inline home.html]: could this be a class?

File: home.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    <div class="container">
+        <p>My name is Caitlin Doyle and I am training to become a full stack web developer at Mayden Academy, training in Bath but based in Bristol.</p>
+    </div>
+</div>
+<div id="featured">
+    <div class="container">
+        <div class="featured-header">
```
Comment (charliecog) [inline home.html]: multiple divs could be changed out for classes

File: home.html [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+            <h2>FEATURED PROJECTS</h2>
+        </div>
+        <div>
+            <div class="featured-project-links featured-project-one col-3 tb-col-2 mb-col-1">
+                <div class="project">
+                    <a href="file:///Users/caitlindoyle/Library/Mobile%20Documents/com~apple~CloudDocs/Documents/Mayden/exercises/formtable%2012022018/index.html" target="_blank"><img src="file:///Users/caitlindoyle/Sites/academy-php7/html/portfolio/images/testform_tile.png" alt="Test Form"></a>
```
Comment (charliecog) [inline home.html]: absolute link

File: home.html [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+            <div class="featured-project-links featured-project-three col-3 tb-col-2 mb-col-1">
+                <div class="project">
+                    <a href="file:///Users/caitlindoyle/Library/Mobile%20Documents/com~apple~CloudDocs/Documents/Mayden/exercises/pilotshop%2014022018/pilotshop.html" target="_blank"><img src="file:///Users/caitlindoyle/Sites/academy-php7/html/portfolio/images/pilotshop_tile.png" alt="Pilot Shop"></a>
+                    <p>Project: PilotShop.<br>First attempt at responsiveness in number of items on a line and hover features.</p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<footer>
```
Comment (charliecog) [inline home.html]: good use of semantic tag but using too many divs - try consolidating using classes

File: css/styles.css [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+}
+
+.logo img {
+    height: 60px;
+}
+
+/*FOOTER */
+
+footer {
+    background-image: url("file:///Users/caitlindoyle/Sites/academy-php7/html/portfolio/images/forestabove.jpg");
```
Comment (charliecog) [inline css/styles.css]: needs relative path

File: css/styles.css [hunk 7]
```diff
@@ -0,0 +1,262 @@
...
+}
+
+.email {
+    float: left;
+    padding-top: 15px;
+}
+
+.media li {
+    float: right;
+    list-style: none;
```
Comment (charliecog) [inline css/styles.css]: navigation list style is also none, best abstracting out common css into its own selector

File: css/styles.css [hunk 8]
```diff
@@ -0,0 +1,262 @@
...
+    .container {
+        width: 85%;
+    }
+
+}
+
+/*PHONE BREAKPOINT*/
+
+@media screen and (max-width: 650px) {
+    body {
```
Comment (charliecog) [inline css/styles.css]: put in whitespace line to match previous media queries for consistency

File: css/styles.css [hunk 9]
```diff
@@ -0,0 +1,262 @@
...
+
+    .media {
+        width: 100%;
+        padding: 0;
+    }
+
+    .media li {
+        width: 20%;
+        float: none;
+        list-style: none;
```
Comment (charliecog) [inline css/styles.css]: same list style on the li

File: about.html [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway">
+    <link rel="stylesheet" type="text/css" href="css/styles.css"/>
+    <link rel="stylesheet" type="text/css" href="css/aboutstyles.css"/>
+</head>
+<body>
+<header class="navbar">
+    <div class="menu">
+        <ul class="navigation">
+            <li>
+                <a href="file:///Users/caitlindoyle/Sites/academy-php7/html/portfolio/home.html">HOME</a>
```
Comment (charliecog) [inline about.html]: relative links

File: about.html [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+            <div class="contact">
+                <a href="file:///Users/caitlindoyle/Sites/academy-php7/html/portfolio/contact.html">CONTACT</a>
+            </div>
+        </div>
+    </div>
+    <div class="logo">
+        <a href="file:///Users/caitlindoyle/Sites/academy-php7/html/portfolio/home.html"><img src="images/CD-logo.png" alt="Logo"/></a>
+    </div>
+</header>
+<div id="banner">
```
Comment (charliecog) [inline about.html]: multiple divs could be replaced by multiple classes

File: about.html [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+    <div class="main">
+        <div class="container">
+            <div class="about-me">
+                <h1>ABOUT ME</h1>
+            </div>
+        </div>
+    </div>
+</div>
+<div id="self-description">
```
Comment (charliecog) [inline about.html]: could be a class

File: about.html [hunk 13]
```diff
@@ -0,0 +1,113 @@
...
+            </div>
+            <div class="about-me">
+                <p>My name is Caitlin Doyle, and I am a full stack web developer in training, training in Bath but living in Bristol.<br><br>In 2017 I finished a Master's degree in Marine Biology. Whilst writing my thesis I had the opportunity to write some code to carry out statistical analyses and found myself really enjoying the process of aiming for a result, working out how to get there, and then writing the code.<br><br>At the end of 2017 I stumbled across Mayden Academy, based in Bath, and really liked what they are doing for training people to become full stack web developers. I applied for the course and was fortunate enough to be offered a place!<br><br>Following the course, I would love to combine these two skillsets of web development and scientific analysis. My goal would be to apply technological solutions to problems facing the planet, as well as making science and research useful and accessible for the wider population.</p>
+            </div>
+        </div>
+    </div>
+</div>
+<div id="skills">
```
Comment (charliecog) [inline about.html]: class

File: about.html [hunk 14]
```diff
@@ -0,0 +1,113 @@
...
+            <div class="skills-logo col-4 lg-tb-col-3 sm-tb-col-2 mb-col-1">
+                <img src="file:///Users/caitlindoyle/Sites/academy-php7/html/portfolio/images/github_logo.svg" alt="GitHub">
+            </div>
+        </div>
+    </div>
+</div>
+<footer>
+    <div class="container">
+        <div class="footer">
+            <div class="hire-link">
```
Comment (charliecog) [inline about.html]: multiple divs could be classes

File: contact.html [hunk 15]
```diff
@@ -0,0 +1,96 @@
...
+            </div>
+        </div>
+    </div>
+    <div class="logo">
+        <a href="home.html"><img src="images/CD-logo.png" alt="Logo"/></a>
+    </div>
+</header>
+<div id="banner">
+    <div class="main">
+        <div class="container">
```
Comment (charliecog) [inline contact.html]: multiple classes

File: contact.html [hunk 16]
```diff
@@ -0,0 +1,96 @@
...
+            <div class="about-me">
+                <h1>CONTACT ME</h1>
+            </div>
+        </div>
+    </div>
+</div>
+<div id="contact-info">
+    <div class="container">
```
Comment (charliecog) [inline contact.html]: multiple classes

File: css/homestyles.css [hunk 17]
```diff
@@ -0,0 +1,144 @@
+.current {
+    background-color: #36494d;
+}
+
+.current a {
+    color: #fdfaf5;
+}
+
+#hero {
+    background-image: url("file:///Users/caitlindoyle/Sites/academy-php7/html/portfolio/images/forestbackground.jpeg");
```
Comment (charliecog) [inline css/homestyles.css]: relative link

File: css/portfoliostyles.css [hunk 18]
```diff
@@ -0,0 +1,66 @@
+
+.current {
```
Comment (charliecog) [inline css/portfoliostyles.css]: repeated from homestyles.css - make generic

File: css/portfoliostyles.css [hunk 19]
```diff
@@ -0,0 +1,66 @@
...
+    background-color: #36494d;
+}
+
+.current a {
```
Comment (charliecog) [inline css/portfoliostyles.css]: repeated from homestyles.css - make generic

File: css/portfoliostyles.css [hunk 20]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+.portfolio-header {
+    text-align: center;
+    font-size: 2rem;
+    margin-bottom: 100px;
+    color: #722a19;
+}
+
+.col-3 {
```
Comment (charliecog) [inline css/portfoliostyles.css]: prob could make generic so can be used in future on new pages

File: index.html [hunk 21]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
```
Comment (charliecog) [inline index.html]: not sure what this file is for? if not used should be deleted. (and home should be renamed to index)

File: portfolio.html [hunk 22]
```diff
@@ -0,0 +1,100 @@
...
+<div id="full-portfolio">
+    <div class="main">
+        <div class="container">
+            <div class="portfolio-header">
+                <h2>PROJECTS</h2>
+            </div>
+            <div>
+                <div class="project-links col-3 tb-col-2 mb-col-1">
+                    <div class="project">
+                        <a href="file:///Users/caitlindoyle/Library/Mobile%20Documents/com~apple~CloudDocs/Documents/Mayden/exercises/formtable%2012022018/index.html" target="_blank"><img src="images/testform_tile.png" alt="Test Form"></a>
```
Comment (charliecog) [inline portfolio.html]: relative links (see elsewhere in this file too)

File: portfolio.html [hunk 23]
```diff
@@ -0,0 +1,100 @@
...
+                        <p>Project: PilotShop.<br>First attempt at responsiveness in number of items on a line and hover features.</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<footer>
+    <div class="container">
+        <div class="footer">
```
Comment (charliecog) [inline portfolio.html]: multiple divs could be multiple classes

File: portfolio.html [hunk 24]
```diff
@@ -0,0 +1,100 @@
...
+                    </div>
+                </div>
+                <div class="project-links col-3 tb-col-2 mb-col-1">
+                    <div class="project">
+                        <a href="file:///Users/caitlindoyle/Library/Mobile%20Documents/com~apple~CloudDocs/Documents/Mayden/exercises/jumbotron%2013022018/mine/jumbotron.html" target="_blank"><img src="images/jumbotron_tile.png" alt="Jumbotron"></a>
+                        <p>Project: Jumbotron.<br>First attempt at page size responsiveness and displaying items inline.</p>
+                    </div>
+                </div>
+                <div class="project-links col-3 tb-col-2 mb-col-1">
+                    <div class="project">
```
Comment (charliecog) [inline portfolio.html]: multiple divs could be multiple classes

File: portfolio.html (hunk 23 above)
Comment (charliecog) [inline portfolio.html]: calling class="footer" inside a footer tag isnt very semantic

File: .gitignore [hunk 25]
```diff
@@ -0,0 +1,3 @@
+*/.DS_Store
+
+
```
Comment (charliecog) [inline .gitignore]: extra whitespace

File: css/styles.css [hunk 26]
```diff
@@ -83,15 +86,37 @@ a {
...
+    background-color: #36494d;
+}
+
+.current a {
+    color: #fdfaf5;
+}
+
+/*GENERIC MAIN STYLING*/
+
+
```
Comment (charliecog) [inline css/styles.css]: extra whitespace

**Comment by charliecog** [comment]
also when you hover over nav link on small browser it covers over the words, just wanted to check if this was a design choice or not?

File: css/styles.css [hunk 27]
```diff
@@ -111,7 +110,7 @@ li {
 
 footer {
     background-color: #9da993;
-    background-image: url("images/forestabove.jpg");
+    background-image: url("../images/forestabove.jpg");
```
Comment (charliecog) [inline css/styles.css]: is this doing anything? where is this displayed?

Comment (ailindoyle) [reply css/styles.css]: this is the background footer image - it vanished when i changed the filepath from absolute to relative, so i put in that background color as a backup, and then worked out that I was missing a bit in my filepath

**Pull request merged**